_layouts/default.html

```diff
@@ -57,6 +57,20 @@
     </div>
 
     {% include scripts.html %}
+    <script>
+      async function getHelmiStatus() {
+        let obj
+            const response = await fetch("https://fiqci-backend-fiqci-workspace.rahtiapp.fi/healthcheck")
+            const result = await response.json();
+            if (result && result.data !== null) {
```

Comment:
You could use optional chaining here `result?.data` (but not necessarily needed)